fcrepo-auth-webac/src/test/resources/spring-test/security.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:context="http://www.springframework.org/schema/context"
+  xmlns:c="http://www.springframework.org/schema/c"
+  xmlns:util="http://www.springframework.org/schema/util"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+ 
+  <context:property-placeholder/>
+    <!-- Shiro config -->
+  
+  <bean id="testAuthFilter" class="org.fcrepo.http.commons.test.util.TestAuthenticationRequestFilter"/>
```

Comment:
Which ticket will tie this configuration into `fcrepo-webapp`?
https://jira.duraspace.org/browse/FCREPO-2753 ?

Reply:
@awoods https://jira.duraspace.org/browse/FCREPO-2763 is the ticket for setting the spring config in fcrepo-webapp.